packages/design-system/src/StatusedSubmitButton.tsx

```diff
@@ -0,0 +1,32 @@
+import { Flex, Spinner, Text } from '@chakra-ui/react';
+import { Maybe } from '@metafam/utils';
+import React, { ReactElement } from 'react';
+
+import { MetaButton } from './MetaButton';
+
+export const StatusedSubmitButton = ({
+  label = 'Submit',
+  status = null,
+  ...props
+}: {
+  label?: Maybe<string>;
+  status?: Maybe<string | ReactElement>;
+}) => (
+  <MetaButton
+    type="submit"
+    _focus={{ filter: 'brightness(1.75)' }}
+    sx={{ ':hover': { filter: 'rotate-hue(-90)', border: '2px solid green' } }}
```

Comment:
```suggestion
    _hover={{ filter: 'rotate-hue(-90)', border: '2px solid green' }}
```

Reply:
Yeah, I got this in #1127. I was trying to figure out why this wouldn't work by using the `sx` attribute rather than `_hover`. It's because  `rotate-hue` is supposed to be `hue-rotate`.